base_layer/core/src/base_node/base_node.rs

```diff
@@ -139,7 +139,7 @@ impl<B: BlockchainBackend> BaseNodeStateMachine<B> {
         let mut state = Starting(states::Starting);
         let mut shared_state = self;
         loop {
-            let _ = shared_state.event_sender.send(state.clone()).await;
+            shared_state.event_sender.send(state.clone()).await;
```

Comment:
Does this not throw a warning that the result is not consumed? We have a lot of warnings.

Reply:
done